pkg/controller/vsphere/reconciler_test.go

```diff
@@ -930,6 +935,31 @@ func TestUpdate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if err := session.WithRestClient(context.TODO(), func(c *rest.Client) error {
+		tagsMgr := tags.NewManager(c)
+
+		id, err := tagsMgr.CreateCategory(context.TODO(), &tags.Category{
+			AssociableTypes: []string{"VirtualMachine"},
+			Cardinality:     "SINGLE",
+			Name:            "CLUSTERID_CATEGORY",
+		})
+		if err != nil {
+			return err
+		}
+
+		_, err = tagsMgr.CreateTag(context.TODO(), &tags.Tag{
+			CategoryID: id,
+			Name:       "CLUSTERID",
+		})
+		if err != nil {
+			return err
+		}
+
+		return nil
+	}); err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
can we cover reconcileTags with a unit `TestReconcileTags`?

Reply:
done

Reply:
thanks!